README.md

```diff
@@ -2,31 +2,378 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Discuss the `String` class
+- Introduce the String Pool and new `String` objects
+- Discuss some of the `String` class' methods
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+The `String` data type is one of the first data types we started using! But up
+until now, we haven't really talked about the `String` class itself. We already
+know that it is an immutable reference type and can be made up of a sequence of
+characters, but can it do more?
 
-## Topic 1
+## The `String` class
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+**The `String` class** has multiple methods for analyzing the contents of
+strings, finding characters, locating substrings, changing the case, and so much
+more! When we start performing these types of methods on `String` data types,
+we typically refer to this as **string manipulation** as we are manipulating the
+`String`.
 
-## Topic 2
+We saw in the last lesson a couple of methods that belong to the `String` class,
+the like constructor that takes a character array and the `toCharArray()`
+method used to convert a `String` into a `char[]`.
+
+Let's look closer at the constructors for the `String` class.
+
+## Constructing Strings
+
+As we have seen in previous lessons, we can initialize a `String` like this:
```

Comment:
Maybe add the definition of "immutable" here in this section? I kind of like this from the immutable classes lesson:

![image](https://user-images.githubusercontent.com/106635778/215212449-01fbaff1-c838-48ce-ac1b-9ea3571ee3e9.png)

Reply:
The introductory character&string lesson was updated to define immutable.